sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 ### Breaking changes
 
 - This version of the SDK defaults to the latest supported service version, which currently is `v1.0`.
+- Renamed method `GetTranslations` to `GetTranslationsStatuses` and same for the async equivalent method.
```

Comment:
Should this be GetAllTranslationStatuses to match GetAllDocumentStatuses?

Reply:
updated